packages/x-components/src/x-modules/search/components/__tests__/redirection.spec.ts

```diff
@@ -0,0 +1,167 @@
+import Vuex, { Store } from 'vuex';
+import { Redirection as RedirectionModel } from '@empathyco/x-types';
+import { createLocalVue, Wrapper, mount } from '@vue/test-utils';
+import { createRedirectionStub } from '../../../../__stubs__';
+import { getDataTestSelector, installNewXPlugin } from '../../../../__tests__/utils';
+import { getXComponentXModuleName, isXComponent } from '../../../../components';
+import { XPlugin } from '../../../../plugins';
+import { RootXStoreState } from '../../../../store';
+import { DeepPartial } from '../../../../utils';
+import Redirection from '../redirection.vue';
+import { resetXSearchStateWith } from './utils';
+
+function renderRedirection({
+  template = `
+  <Redirection :mode="mode" :delay="delay"  v-slot="{ redirection, redirect, abortRedirect }">
+     <span data-test="redirection-url">{{ redirection.url }}</span>
+     <button data-test="redirection-accept" @click="redirect">Redirect now!</button>
+     <button data-test="redirection-abort" @click="abortRedirect">Abort redirection!</button>
+  </Redirection>
+`,
+  redirections = [createRedirectionStub('redirection')],
+  mode = 'auto',
+  delay = 1
+}: RenderRedirectionOptions = {}): RenderRedirectionAPI {
+  const localVue = createLocalVue();
+  localVue.use(Vuex);
+  const store = new Store<DeepPartial<RootXStoreState>>({});
+  installNewXPlugin({ store }, localVue);
+
+  XPlugin.resetInstance();
+  resetXSearchStateWith(store, { redirections });
+  const wrapper = mount(
+    {
+      components: {
+        Redirection
+      },
+      template
+    },
+    {
+      data() {
+        return {
+          mode,
+          delay
+        };
+      },
+      localVue,
+      store
+    }
+  );
+
+  const redirectionWrapper = wrapper.findComponent(Redirection);
+
+  return {
+    wrapper,
+    redirectionWrapper,
```

Comment:
```suggestion
    wrapper: redirectionWrapper,
```

Reply:
If you do this, you can remove the redirectionWrapper from the RenderRedirectionAPI, because right now you are returning 2 times the same thing.